code/__HELPERS/icons.dm

```diff
@@ -872,6 +872,52 @@ The _flatIcons list is a cache for generated icon files.
 	del(atom_icon)
 	return text_image
 
+// Pick a random animal instead of the icon, and use that instead
+/proc/getRandomAnimalImage(atom/A)
+	var/list/icons = list()
+	var/icon/selected
+
+	icons += new/icon('icons/mob/animal.dmi', "cow")

```

Comment:
Yeah, but then we wouldn't be able to pick Ian in a wheelchair. If we're concerned about the time creating the icons will take, I can probably ask for advice on making the list global so it only needs to be created once.